src/AutoRest.CSharp/AutoRest.CSharp.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.AutoRest.CSharp</PackageId>
     <OutputType>Exe</OutputType>
     <Version>3.0.0</Version>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Don't we want to update this for [AutoRest.CodeModel.csproj](4647f702bf90e43ae34d7bfd71a568673895f3d5), [AutoRest.TestServer.Tests.csproj](https://github.com/Azure/autorest.csharp/blob/4647f702bf90e43ae34d7bfd71a568673895f3d5/test/AutoRest.TestServer.Tests/AutoRest.TestServer.Tests.csproj#L4) and for [this](https://github.com/Azure/autorest.csharp/blob/4647f702bf90e43ae34d7bfd71a568673895f3d5/src/AutoRest.CSharp/build/CodeGeneration.targets#L32) command?